test/declarations.ts

```diff
@@ -69,31 +71,38 @@ Spec.prototype.expectAppError = function <ErrorCode extends ERROR_CODE>(
     error: (args as string[]).reduce((arg, extra) => arg.replaceAll('%', extra), ErrorData[errorCode].message),
   });
 };
-Spec.prototype.expectUe = function (ue: FakeUe, rates: Array<{ criterionId: string; value: number }> = []) {
+Spec.prototype.expectUe = function (ue: FakeUe, rates: Array<{ criterionId: string; value: number }>) {
   return (<Spec>this).expectStatus(HttpStatus.OK).expectJsonMatchStrict(
     deepDateToString({
-      ...omit(ue, 'id', 'validationRate', 'createdAt', 'updatedAt', 'openSemesters'),
-      name: getTranslation(ue.name, this.language),
-      info: {
-        ...omit(ue.info, 'id'),
-        objectives: getTranslation(ue.info.objectives, this.language),
-        comment: getTranslation(ue.info.comment, this.language),
-        program: getTranslation(ue.info.program, this.language),
-      },
-      workTime: omit(ue.workTime, 'id', 'ueId'),
-      credits: ue.credits.map((credit) => omit(credit, 'id', 'ueId', 'categoryId')),
-      branchOption: ue.branchOption.map((branchOption) => ({
-        ...pick(branchOption, 'code', 'name'),
-        branch: pick(branchOption.branch, 'code', 'name'),
-      })),
-      openSemester: ue.openSemesters
-        .mappedSort((semester) => semester.start.toISOString())
-        .map((semester) => ({
-          ...semester,
-          start: semester.start.toISOString(),
-          end: semester.end.toISOString(),
-        })),
-      starVotes: Object.fromEntries(rates.map((rate) => [rate.criterionId, rate.value])),
+      code: ue.code,
+      creationYear: ue.creationYear,
+      updateYear: ue.updateYear,
+      ...(rates ? { starVotes: Object.fromEntries(rates.map((rate) => [rate.criterionId, rate.value])) } : {}),
+      ofs: [
```

Comment:
je crois que je l'avais dit plus haut, mais je suis toujours pour appeler ça ueofs, pour réduire le nombre d'acronymes (surtout que of c'est un mot en anglais, du coup c'est un peu perturbant au singulier XD)